source/locale/tr/gestures.ini

```
@@ -3,6 +3,8 @@
 	rightMouseClick       = kb(laptop):NVDA+ü
 	toggleLeftMouseButton       = kb(laptop):NVDA+control+ğ
 	toggleRightMouseButton       = kb(laptop):NVDA+control+ü
+	navigatorObject_previousInFlow =kb(laptop):NVDA++shift+ğ
+	navigatorObject_nextInFlow =kb(laptop):NVDA+shift+ü
```

I don't think there should be two `+`s

```suggestion
	navigatorObject_previousInFlow = kb(laptop):NVDA+shift+ğ
	navigatorObject_nextInFlow = kb(laptop):NVDA+shift+ü
```